src/components/Inputs/inputs.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+
+const monthList = [
+  {id: "1", month:"January"},
+  {id: "2", month:"February"},
+  {id: "3", month:"March"},
+  {id: "4", month:"April"},
+  {id: "5", month:"May"},
+  {id: "6", month:"June"},
+  {id: "7", month:"July"},
+  {id: "8", month:"August"},
+  {id: "9", month:"September"},
+  {id: "10", month:"October"},
+  {id: "11", month:"November"},
+  {id: "12", month:"December"}
+];
+
+const ownersList = [{id: "1", name:"Alex"},{id: "2", name:"Luis"}];
+const yearList = [{id: "1", year:"2019"},{id: "2", year:"2018"}];
+
+
+class Inputs extends Component {
```

Comment:
If this component won't have state, remember that you can change it to a `stateless component` :)

You could end up having something like 
```
import React from 'react'

const Input = () => {

}
```

Also, try naming your components on `singular`

Reply:
Got it